driver/makedepend.ml

```diff
@@ -574,10 +574,11 @@ let print_version_num () =
 
 let main () =
   Clflags.classic := false;
-  add_to_list first_include_dirs Filename.current_dir_name;
   Compenv.readenv ppf Before_args;
   Clflags.reset_arguments (); (* reset arguments from ocamlc/ocamlopt *)
   Clflags.add_arguments __LOC__ [
+     "-exclude-cur-dir", Arg.Set Clflags.exclude_dir,
+        " Exclude the current directory in searched directories";
```

Comment:
The term `searched directories` does not appear in the rest of the documentation. It is better to try to stay as coherent as possible. Currently, we have:

>  -I <dir>  Add <dir> to the list of include directories

Reply:
`Exclude the current directory in list of include directories` would make a better desc ?

Reply:
`Exclude` may suggest that we are actively removing `.` even if it is added with `-I .`. It is better to tell that we are not implicitly adding `.` to the list of included directories.